data/sql/grid_alternatives.sql

```diff
@@ -144,6 +144,11 @@ VALUES
 ('dvr90_2002.tif','dk_sdfi_dvr90_2002.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dvr90_2002.tif',1,1,NULL),
 ('dvr90_2013.tif','dk_sdfi_dvr90_2013.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dvr90_2013.tif',1,1,NULL),
 ('dvr90_2023.tif','dk_sdfi_dvr90_2023.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dvr90_2023.tif',1,1,NULL),
+('dklat_2022.tif','dk_sdfi_dklat_2022.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dklat_2022.tif',1,1,NULL),
+('dklat_2023.tif','dk_sdfi_dklat_2023.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dklat_2023.tif',1,1,NULL),
+('dkmsl_2022.tif','dk_sdfi_dkmsl_2022.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dkmsl_2022.tif',1,1,NULL),
+('dkmsl_2023.tif','dk_sdfi_dkmsl_2023.tif',NULL,'GTiff','hgridshift',0,NULL,'https://cdn.proj.org/dk_sdfi_dkmsl_2023.tif',1,1,NULL),
```

Comment:
AFAICS, those 4 should be geoid_like

Reply:
Yeah. Actually I just copied and modified the above DVR90-lines, so those need adjusting as well